data/models/mongooseResolvers.js

```diff
@@ -0,0 +1,61 @@
+// This file has resolvers for queries that get data from MongoDB using Mongoose.
+
+import { mongoPublicationModel }  from './mongooseSchema.js';
+
+// Function to get the first paper in the collection.
+const getPublication = () => {
+  console.log('Getting one publication from DB');
+  return new Promise((resolve, reject) => {
+    mongoPublicationModel.findOne({}, (err, record) => {
```

Comment:
Does `findOne` return a promise if not passed a callback? If so `new Promise(...)` would be unnecessary and this file would look a lot cleaner.

Reply:
findOne doesn't return a promise directly. We should either use it with exec() or new Promise().